packages/mdc-menu/simple/foundation.js

```diff
@@ -83,6 +83,7 @@ export default class MDCSimpleMenuFoundation extends MDCFoundation {
     this.scaleY_ = 0;
     this.running_ = false;
     this.selectedTriggerTimerId_ = 0;
+    this.animationRequestId_ = null;
```

Comment:
After running some tests on multiple browsers, it seems safe to call `cancelAnimationFrame(0)`, so making suggested changes.